tools/cldr-apps/src/main/java/org/unicode/cldr/web/api/GenerateVxml.java

```diff
@@ -134,4 +136,59 @@ public static final class VxmlResponse {
         @Schema(description = "Verification warning messages")
         public String[] verificationWarnings;
     }
+
+    @GET
+    @Path("/download")
+    @Produces(MediaType.APPLICATION_OCTET_STREAM)
```

Comment:
I also tested with `MediaType.APPLICATION_JSON` here, and it still worked; likewise with "application/json" instead of "application/zip" below